File: app/src/main/java/com/formichelli/dfnightselfies/takephotointent/DFNightSelfiesFromIntentActivity.java
package com.formichelli.dfnightselfies.takephotointent;

import android.os.Bundle;

import com.formichelli.dfnightselfies.DFNightSelfiesMainActivity;
import com.formichelli.dfnightselfies.WindowUtil;

public class DFNightSelfiesFromIntentActivity extends DFNightSelfiesMainActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        WindowUtil.setupWindow(this);

        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, new DFNightSelfiesFromIntentFragment())
                .commit();
    }
}

File: app/src/main/java/com/formichelli/dfnightselfies/takephotointent/DFNightSelfiesFromIntentActivity.kt
package com.formichelli.dfnightselfies.takephotointent

import android.os.Bundle

import com.formichelli.dfnightselfies.DFNightSelfiesMainActivity
import com.formichelli.dfnightselfies.util.Util

class DFNightSelfiesFromIntentActivity : DFNightSelfiesMainActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        Util.setupWindow(this)

        fragmentManager.beginTransaction()
                .replace(android.R.id.content, DFNightSelfiesFromIntentFragment())
                .commit()
    }
}
